model-optimizer/extensions/middle/AddMeanScaleValues.py

```diff
@@ -52,6 +52,16 @@ def insert_pre_processing(graph: Graph, input_node: Node, node_mean_scale_values
         name = input_node.soft_get('name', input_node.id) + '/' + preprocessing_name
         preprocessing = create_op_with_const_inputs(graph, op=op, port_value_dict={1: value}, op_attrs={'name': name})
 
+        if input_node.has_port('out', 0) and len(input_node.out_port(0).get_destinations()) == 1:
```

Comment:
please use `input.is_out_port_connected(0)` instead of `input_node.has_port('out', 0)`

Reply:
Done.